integration/src/integration/java/pl/allegro/tech/hermes/integration/setup/HermesManagementInstance.java

```diff
@@ -89,6 +92,7 @@ public HermesManagementInstance start() {
         }
 
         private void waitUntilStructureInZookeeperIsCreated(CuratorFramework zookeeper) {
+            logger.info("Waiting for zookeeper structure to be created");
```

Comment:
Can you add a log when waiting for 120 seconds results positively (and negatively too)?